### PR TITLE
introduce a display method in Morph and Morph class to reduce the num…

### PR DESCRIPTION
**Pull request by Ducasse**
…ber of direct reference to one of our last globals

I doubt that it will work but let us see.

### CODE REVIEW

**Comment by Ducasse** [comment]
I will rename display in displayInstance because display is already defined to display the receiver in the class DisplayObject.

**Comment by Ducasse** [comment]
And I would love to rename DisplayObject into DisplayableObject

File: src/Athens-Examples/AthensTigerWindow.class.st [hunk 1]
```diff
@@ -4,6 +4,7 @@ I'm an example that shows a tiger that moves via rotation, translation and zoom
 Class {
 	#name : 'AthensTigerWindow',
 	#superclass : 'StandardWindow',
+	#classTraits : '{} + TraitedClass',
```
Comment (guillep) [inline src/Athens-Examples/AthensTigerWindow.class.st:7]: Hmm, we should fix this, this is the default class trait

Comment (guillep) [reply src/Athens-Examples/AthensTigerWindow.class.st:7]: (not in this PR)

**Review by guillep — APPROVED** [review]
Looks ok, now I really ask myself why are so many users also.

**Comment by Ducasse** [comment]
```
MetacelloNotification: Loaded -> MenuRegistration-tonel.1 --- tonel:///builds/workspace/est_and_branch_Pipeline_PR-15859/64/src --- cache
MetacelloNotification: Loading -> Fuel-Core-tonel.1 --- tonel:///builds/workspace/est_and_branch_Pipeline_PR-15859/64/src --- cache
MetacelloNotification: Loaded -> Fuel-Core-tonel.1 --- tonel:///builds/workspace/est_and_branch_Pipeline_PR-15859/64/src --- cache
MetacelloNotification: Starting atomic load
Error: The following definitions had errors while loading.  Press Proceed to try to load them again (they may work on a second pass):
  AlphaImageMorph
  ThreePhaseButtonMorph
...
```

I would love to know how we can access to the errors.

File: src/Athens-Examples/AthensTigerWindow.class.st (hunk 1 above)
Comment (Ducasse) [reply src/Athens-Examples/AthensTigerWindow.class.st:7]: Do you have an idea where it comes from because I did not touch this part.

Comment (guillep) [reply src/Athens-Examples/AthensTigerWindow.class.st:7]: Yes, it's the fix we did the other day to Traits

Comment (Ducasse) [reply src/Athens-Examples/AthensTigerWindow.class.st:7]: Yes I was connecting after...:)

**Comment by Ducasse** [comment]
I think that I will need some help. And I also wonder why so many domain objects (outside of this PR) needs to access Display.

**Comment by Ducasse** [comment]
I close it because we redid it differently.

**Pull request closed without merging**